website/src/user-guide/output-format.md

```diff
@@ -63,4 +63,15 @@ To make the text really clickable in your terminal you might need to register th
 
 If you run PHPStan analysis within Docker container (or using other virtualization tools) you may need to use `%relFile%` instead of `%file%`. This will use file's path relative to the current working directory. In the end your `editorUrl` should look like this: `phpstorm://open?file=/path/to/your/project/%%relFile%%&line=%%line%%`.
 
+
+<div class="text-xs inline-block border border-green-600 text-green-600 bg-green-100 rounded px-1 mb-4">Available in PHPStan 1.9.3</div>
+
+You may also want to change the default title of the clickable link to contain line to be able to quickly copy-paste it to your IDE when used within an environment that is not clickable (like CI output). Here is how:
+```neon
+parameters:
+	editorUrlTitle: '%%relFile%%:%%line%%'
+```
+
+--------------------
```

Comment:
I think the last paragraph should not be considered as 'available since 1.9.3', so I added this line.